.circleci/config.yml

```
@@ -10,7 +10,14 @@ jobs:
       - run:
           name: "Build COMPASS docker image"
           command: |
-            docker build . -f docker/Dockerfile
+            docker build . -f docker/Dockerfile -t test_image
+      - run:
+          name: "Run integration tests"
+          command: |
+            docker run test_image bash -c "cd ~/OPERA/COMPASS &&\
```

```suggestion
            docker run test_image bash -ci "cd ~/OPERA/COMPASS &&\
```

This is what we do in the isce3 docker commands. The `-i` flag forces an interactive prompt so the .bashrc is sourced properly. Hopefully that'll fix the issues with the conda environment not being activated properly